index.html

```diff
@@ -8,4 +9,4 @@
 <body>
     <p>Hello World. Where is the rice?</p>
```

Comment:
All of this is incorrect.

Reply:
I will examine and decide to follow instruction.